src/sidecar/logging.ts

```diff
@@ -36,7 +115,12 @@ export function startTailingSidecarLogs(): Tail | undefined {
 
   let logTailer: Tail;
   try {
-    logTailer = new Tail(sidecarLogfilePath);
+    // https://github.com/lucagrulla/node-tail/blob/master/README.md#constructor-parameters
+    logTailer = new Tail(sidecarLogfilePath, {
+      useWatchFile: true,
```

Comment:
This is the main part that _may_ help with #567 based on the [docs](https://github.com/lucagrulla/node-tail/blob/master/README.md#constructor-parameters):
> `useWatchFile`: if set to true will force the use of `fs.watchFile` over delegating to the library the choice between `fs.watch` and `fs.watchFile` (default: `false`).